feature/networkinstance/management_vrf/README.md

```diff
@@ -0,0 +1,85 @@
+    *   Configure the following interfaces are in the `mgmt_vrf_name`VRF:
+        *   lo0.100, which is configured with only the IPv6 management IP that’s extracted from `lab_infra_loopback_name`. No IPv4 address is configured under lo0.100.
+        *   Port-1, port-2, port-11, port-12 and OOB interfaces.
+    *   Configure the following services to listen on the lo0.100 management IP.
+        *   gRIBI
+        *   gNOI
+        *   gNMI
+        *   P4RT
+    *   Configure the following services with the expectation that lo0.100 IP will be used for communication.
+        *   sFLOW
+        *   Syslog
+        *   TACACS services.
+        *   SSH services.
+        *   Specify the target name server.
+        *   Specify the target NTP server.
+    *   Configure two `::/0` static routes configured in the management VRF, with default route preference and next hops being the adjacent IP of port-11 and port-12. This will be the primary channel for management traffic. In prod, this will be BGP routes. However, we don’t have to enforce this on the lab infrastructure.
+    *   Configure a `::/0` static route configured in the management VRF with route preference 220 going via OOB interface. This will be the backup channel for management traffic.
+*   Configure a v6 BGP between DUT port-1 and ATE port-1.
+*   Configure a v6 BGP between DUT port-2 and ATE port-2.
+*   Configure ATE to advertise a prefix (say, [2001:DB8:0::/64](https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#ipv6)) to DUT.
+
+## Tests
+
+### Test-1, routing protocol validation
+
+Via gNMI, validate that:
+
+*   The configured 2 BGP sessions are up and receive the advertised `2001:DB8:0::/64/126` route on DUT.
+*   The `::/0` static route is of the expected route preference value.
+
+### Test-2, service available via linecard interfaces
+
+1.  Via gNMI, shutdown all OOB ports so that port-11 and port-12 are the only ingress ports to manage the device. This step itself validates the gNMI service availability.
+2.  Issue a gRIBI Flush() and expect a successful response.
+3.  Issue a gNOI Ping() test to ping itself and expect a successful response.
+4.  Issue a P4RT `MasterArbitrationUpdate` with random `device_id`, and expect a `StreamMessageResponse` response (no need to check the correctness).
+5.  sFLOW will be tested in its own feature profile test.
+6.  Use Ondatra raw API to run CLI command to refresh the time via NTP.  This also validates that the SSH is working.
+7.  Use Ondatra raw API to run CLI command to lookup google.com.
+8.  Syslog should be tested in its own feature profile test.
+9.  Unshut OOB ports
+
+### Test-3, service available after CONTROLLER CARD switchover
+
+1.  Via gNMI, shutdown all OOB ports so that port-11 and port-12 are the only ingress ports to manage the device.
```

Comment:
what about existing gNOI-3.3: control_card switchover?

Reply:
I think gNOI-3.3 is more about the switch action, but here we are add on top of it that
* we have management VRF.
* we want to check service status.